New section for dummy endpoints (#10281)
friendica/friendica: doc/API-Mastodon.md

## Currently unimplemented endpoints

These emdpoints are planned to be implemented somewhere in the future.

- [`PATCH /api/v1/accounts/update_credentials`](https://docs.joinmastodon.org/methods/accounts/)
- [`GET /api/v1/instance/activity`](https://docs.joinmastodon.org/methods/instance#weekly-activity)

## Dummy endpoints

> These endpoints are returning empty data to avoid error messages with (for example) mobile clients.

Reviewer: ```suggestion
These endpoints are returning empty data to avoid error messages with some mobile clients.
```

Author: Its more "nearly all" and also it can affect desktop apps as well. So "some" isn't the right word here.

They belong to a functionality that doesn't exist in Friendica yet.

- [`GET /api/v1/accounts/:id/identity_proofs`](https://docs.joinmastodon.org/methods/accounts/)
- [`GET /api/v1/announcements`](https://docs.joinmastodon.org/methods/announcements/)
- [`GET /api/v1/endorsements`](https://docs.joinmastodon.org/methods/accounts/endorsements/)
- [`GET /api/v1/filters`](https://docs.joinmastodon.org/methods/accounts/filters/)
- [`GET /api/v1/markers`](https://docs.joinmastodon.org/methods/timelines/markers/)
- [`GET /api/v1/scheduled_statuses`](https://docs.joinmastodon.org/methods/statuses/scheduled_statuses/)

## Non supportable endpoints

These endpoints won't be implemented at the moment.

> They belong to a functionality that doesn't exist in Friendica yet.

Reviewer: ```suggestion
They refer to features that don't exist in Friendica yet.
```
